From: Robert Rostohar <Robert.Rostohar@arm.com>
Date: Mon, 21 Dec 2020 16:33:25 +0100
Subject: [PATCH 1/3] Remove non-portable platform headers

Remove inclusion of non-portable platform headers:
 - device_cfg.h: Device specific configuration (USART baudrate)
 - target_cfg.h: Target specific configuration (USART driver)

TF-M configuration file is used to specify user settings
(USART driver and baudrate, ...).

Update inclusion of CMSIS USART driver:
 - #include "Driver_USART.h (registered path)
---
 app/main_ns.c | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/app/main_ns.c b/app/main_ns.c
index c52a859..9b63507 100644
--- a/app/main_ns.c
+++ b/app/main_ns.c
@@ -16,10 +16,8 @@
 #ifdef PSA_API_TEST_NS
 #include "psa_api_test.h"
 #endif
-#include "target_cfg.h"
 #include "tfm_plat_ns.h"
-#include "driver/Driver_USART.h"
-#include "device_cfg.h"
+#include "Driver_USART.h"
 #ifdef TFM_MULTI_CORE_TOPOLOGY
 #include "tfm_multi_core_api.h"
 #include "tfm_ns_mailbox.h"
-- 
